PreFreeSurfer/PreFreeSurferPipeline.sh

```diff
@@ -208,11 +211,11 @@ opts_AddMandatory '--brainsize' 'BrainSize' 'size_value' "Brain size estimate in
 
 opts_AddMandatory '--fnirtconfig' 'FNIRTConfig' 'file_path' "FNIRT 2mm T1w Configuration file"
 
-opts_AddOptional '--fmapmag' 'MagnitudeInputName' 'file_path' "Siemens/Philips Gradient Echo Fieldmap magnitude file"
+opts_AddOptional '--fmapmag' 'MagnitudeInputName' 'file_path' "Siemens/Philips/GE HealthCare Gradient Echo Fieldmap magnitude file"
 
-opts_AddOptional '--fmapphase' 'PhaseInputName' 'file_path' "Siemens/Philips Gradient Echo Fieldmap phase file"
+opts_AddOptional '--fmapphase' 'PhaseInputName' 'file_path' "Siemens/Philips Gradient Echo Fieldmap phase file or GE HealthCare Fieldmap in Hertz"
 
-opts_AddOptional '--fmapgeneralelectric' 'GEB0InputName' 'file_path' "General Electric Gradient Echo Field Map file  Two volumes in one file  1. field map in deg  2. magnitude"
+opts_AddOptional '--fmapgeneralelectric' 'GEB0InputName' 'file_path' "GE HealthCare Legacy Gradient Echo Field Map file. Two volumes in one file  1. field map in hertz  2. magnitude"
```

Comment:
```suggestion
opts_AddOptional '--fmapgeneralelectric' 'GEB0InputName' 'file_path' "GE HealthCare Legacy Gradient Echo Field Map approach, which contains two volumes in one file:  1. field map in hertz;  2. magnitude image"
```

Reply:
Also, should we perhaps change this flag to `--fmapgelegacy` as the forward-facing name, with `--fmapgeneralelectric` still allowed as an alternative? (I think the library allows specification of alternative argument flags, right @coalsont ?)

Reply:
No problem to change `--fmapgeneralelectric` in `--fmapgelegacy`. 
Having the 2 possibilities might be confusing, no ? and complicate maintenance.

Reply:
I guess since the old approach no longer even exists conceptually, it makes sense to drop `--fmapgeneralelectric` as a flag entirely.  Anyone trying to use it (e.g., with existing scripts) will get any error anyway that `DeltaTE` doesn't exist.   @coalsont: do you agree?

Reply:
Yes, providing one or more legacy flags after the default value (quoted empty string if you want the same behavior as no default value) will take it as another way to do the same thing (but it is not mentioned in the help info).  If we don't want to accept `--fmap`, then there is no need for a legacy flag here.  I think we can/should take `--fmap` here, and, per below, if we wanted to change the name at all, the flag should be brand-agnostic, such as `--fmapcombined`, even if only one brand used it so far.

I don't think any of the flags should contain brand names, they should simply be used for handling different arrangements of the input data.

One of the two new methods does take data arranged in the `--fmap` format.